code/LINDA/LINDA_turf_tile.dm

```diff
@@ -207,7 +207,23 @@
 				if(excited_group)
 					last_share_check()
 
-	air.react()
+	if(air)  //How did this even break?
```

Comment:
Could you explain what that chart means? Also, what leads you to believe that this, specifically, is bringing the server down?

I think the point Tigercat is trying to make is that unless there's definitive proof that this is what is truly bringing the server down, we want this to runtime, as air being `null` on a turf is indicative of a much larger problem, and the code here is masking the problem (why air is null to begin with).

It's kinda like if a human mob had a `null` species; if that ever happens, it shouldn't be painted over with a `if(species)` check, but rather the underlying problem of why a human mob isn't getting a species to begin with.